detox/src/client/Client.test.js

```diff
@@ -676,7 +676,7 @@ describe('Client', () => {
   });
 
   describe('on AppWillTerminateWithError', () => {
-    it('should schedule the app termination in 5 seconds, and reject pending', async () => {
+    it('should schedule the app termination in 5 seconds, and reject pending and future requests until the app reconnects', async () => {
```

Comment:
Doesn't this justify splitting to 2 separate tests, as with the prod-tests equivalent?